client/Lykke.Service.BlockchainWallets.Client/BlockchainWalletsFailureHandler.cs

```diff
@@ -8,10 +11,19 @@ namespace Lykke.Service.BlockchainWallets.Client
     public class BlockchainWalletsFailureHandler: IBlockchainWalletsFailureHandler
     {
         private readonly Policy _circuitBreakPolicy;
+        private readonly ISet<HttpStatusCode> _statusCodesToBreakCircuit;
 
         public BlockchainWalletsFailureHandler(TimeSpan durationOfBreak)
         {
             _circuitBreakPolicy = BuildCircuitBreakerPolicy(durationOfBreak);
+
+            _statusCodesToBreakCircuit = new[]
+            {
+                HttpStatusCode.InternalServerError,
+                HttpStatusCode.BadGateway,
+                HttpStatusCode.GatewayTimeout,
+                HttpStatusCode.ServiceUnavailable
```

Comment:
Add HttpStatusCode.RequestTimeout to make responses list as here https://github.com/LykkeCity/Lykke.Service.BlockchainApi.Client/blob/master/src/Lykke.Service.BlockchainApi.Client/ApiRunner.cs#L106-L110